src/boundingbox.ts

```diff
@@ -3,58 +3,76 @@
 //
 // Copyright Mohit Muthanna 2010
 
+import { RenderContext } from './types/common';
+
 // Bounding boxes for interactive notation
 
 export class BoundingBox {
-  static copy(that) {
+  protected y: number;
+  protected x: number;
+  protected w: number;
+  protected h: number;
```

Comment:
Why not private for all these?

Reply:
I thought we have decided to use `protected` in general. `private` only when `get/set` functions make a special handling like with `context`. This is here not the case and the class has also no child.

Reply:
Okay, that's fine.